relay-general/src/store/transactions/processor.rs

```diff
@@ -276,16 +278,25 @@ fn set_default_transaction_source(event: &mut Event) {
 /// Replaces UUIDs, SHAs and numerical IDs in transaction names by placeholders.
 /// Returns `Ok(true)` if the name was changed.
 fn scrub_identifiers(string: &mut Annotated<String>) -> Result<bool, ProcessingAction> {
-    let capture_names = TRANSACTION_NAME_NORMALIZER_REGEX
-        .capture_names()
-        .flatten()
-        .collect::<Vec<_>>();
+    scrub_identifiers_with_regex(string, &TRANSACTION_NAME_NORMALIZER_REGEX)
+}
+
+/// Normalize the given SQL-query-like string.
+fn scrub_sql_queries(string: &mut Annotated<String>) -> Result<bool, ProcessingAction> {
+    scrub_identifiers_with_regex(string, &SQL_NORMALIZER_REGEX)
+}
+
+fn scrub_identifiers_with_regex(
+    string: &mut Annotated<String>,
+    regex: &Lazy<Regex>,
```

Comment:
Can we not call it `regex` ? Since this is the name of the crate as well, which can be confusing. 
Maybe something like 
```suggestion
    re: &Lazy<Regex>,
```
Which usually the way regexes called in the code. Or `pattern` also will do.